docs/standard/serialization/binary-serialization-core.md

```diff
@@ -0,0 +1,127 @@
+- <xref:System.Data.SqlTypes.SqlString?displayProperty=fullName>   
+- <xref:System.DateTime?displayProperty=fullName>   
+- <xref:System.DateTimeOffset?displayProperty=fullName>   
+- <xref:System.Decimal?displayProperty=fullName>   
+- <xref:System.Double?displayProperty=fullName>   
+- <xref:System.Drawing.Color?displayProperty=fullName>   
+- <xref:System.Drawing.Point?displayProperty=fullName>   
+- <xref:System.Drawing.PointF?displayProperty=fullName>   
+- <xref:System.Drawing.Rectangle?displayProperty=fullName>   
+- <xref:System.Drawing.RectangleF?displayProperty=fullName>   
+- <xref:System.Drawing.Size?displayProperty=fullName>   
+- <xref:System.Drawing.SizeF?displayProperty=fullName>   
+- <xref:System.Enum?displayProperty=fullName>   
+- <xref:System.Exception?displayProperty=fullName>   
+- <xref:System.Globalization.CompareInfo?displayProperty=fullName>   
+- <xref:System.Globalization.SortVersion?displayProperty=fullName>   
+- <xref:System.Guid?displayProperty=fullName>   
+- <xref:System.Int16?displayProperty=fullName>   
+- <xref:System.Int32?displayProperty=fullName>   
+- <xref:System.Int64?displayProperty=fullName>   
+- <xref:System.IntPtr?displayProperty=fullName>   
+- <xref:System.Net.Cookie?displayProperty=fullName>   
+- <xref:System.Net.CookieCollection?displayProperty=fullName>   
+- <xref:System.Net.CookieContainer?displayProperty=fullName>   
+- <xref:System.Net.PathList?displayProperty=fullName>   
+- <xref:System.Nullable%601?displayProperty=fullName>   
+- <xref:System.Numerics.BigInteger?displayProperty=fullName>   
+- <xref:System.Numerics.Complex?displayProperty=fullName>   
+- <xref:System.Object?displayProperty=fullName>   
+- <xref:System.SByte?displayProperty=fullName>   
+- <xref:System.Single?displayProperty=fullName>   
+- <xref:System.String?displayProperty=fullName>   
+- <xref:System.StringComparer?displayProperty=fullName>   
+- <xref:System.Text.StringBuilder?displayProperty=fullName>   
+- <xref:System.TimeSpan?displayProperty=fullName>   
+- <xref:System.TimeZoneInfo?displayProperty=fullName>   
+- <xref:System.Tuple?displayProperty=fullName>   
+- <xref:System.UInt16?displayProperty=fullName>   
+- <xref:System.UInt32?displayProperty=fullName>   
+- <xref:System.UInt64?displayProperty=fullName>   
+- <xref:System.UIntPtr?displayProperty=fullName>   
+- <xref:System.Uri?displayProperty=fullName>   
+- <xref:System.ValueTuple?displayProperty=fullName>   
+- <xref:System.ValueType?displayProperty=fullName>   
+- <xref:System.Version?displayProperty=fullName>   
+- <xref:System.WeakReference?displayProperty=fullName>   
+- <xref:System.WeakReference%601?displayProperty=fullName>   
+
+## Remarks
+
+As the nature of binary serialization allows the modification of private members inside an object and therefore changing the state of it, other serialization frameworks like JSON.NET which operate on the public API surface are recommended.
```

Comment:
My suggestion was to move as a subsection with that title within https://github.com/dotnet/docs/blob/master/docs/standard/serialization/binary-serialization.md. Do you need an individual topic for this @ViktorHofer?

No worries! My pleasure.